build_ngtf.py

```diff
@@ -339,21 +339,14 @@ def main():
 
         # Now build OpenVINO
         openvino_cmake_flags = [
-            "-DENABLE_V10_SERIALIZE=ON",
-            "-DENABLE_TESTS=OFF",
-            "-DENABLE_SAMPLES=OFF",
-            "-DENABLE_FUNCTIONAL_TESTS=OFF",
-            "-DENABLE_VPU=OFF",  # TODO: Fix OpenVINO VPU build
-            "-DNGRAPH_ONNX_IMPORT_ENABLE=OFF",
-            "-DNGRAPH_TEST_UTIL_ENABLE=OFF",
-            "-DNGRAPH_USE_CXX_ABI=" + cxx_abi,
+            "-DENABLE_V10_SERIALIZE=ON", "-DENABLE_TESTS=OFF",
+            "-DENABLE_SAMPLES=OFF", "-DENABLE_FUNCTIONAL_TESTS=OFF",
+            "-DENABLE_VPU=ON", "-DNGRAPH_ONNX_IMPORT_ENABLE=OFF",
+            "-DNGRAPH_TEST_UTIL_ENABLE=OFF", "-DNGRAPH_USE_CXX_ABI=" + cxx_abi,
```

Comment:
Does the code formatter do this? One per line is easier to read here.

Reply:
Yes, I edited and then formatter overwrote